src/others/helpers/map-utils.test.ts

```diff
@@ -1,264 +1,59 @@
+  });
 });
 
-test("groupLocationGroupByDate", () => {
-  const locationGroup: DecodedAidRequestGroupedByLocation = {
-    //@ts-ignore
-    location: { name: "Zaliznychne", lat: 47.645, lon: 36.169 },
-    total: 159,
-    decodedAidRequests: [
-      { date: "2022-03-11", amount: 10, name: "Sanitary pads" },
-      { date: "2022-03-12", amount: 4, name: "Personal hygiene kits" },
-      { date: "2022-03-11", amount: 15, name: "Masks" },
-      { date: "2022-03-11", amount: 19, name: "Baby Products" },
-      { date: "2022-03-11", amount: 16, name: "Medical Kits / Supplies" },
-      { date: "2022-03-11", amount: 10, name: "Torches" },
-      { date: "2022-03-11", amount: 11, name: "Water" },
-      { date: "2022-03-11", amount: 21, name: "Personal hygiene kits" },
-      { date: "2022-03-11", amount: 20, name: "Tampons" },
-      { date: "2022-03-11", amount: 10, name: "Food" },
-      { date: "2022-03-12", amount: 4, name: "Medical Kits / Supplies" },
-      { date: "2022-03-12", amount: 4, name: "Baby Products" },
-      { date: "2022-03-11", amount: 15, name: "Baby Food" },
-    ],
-  };
-
-  expect(groupLocationGroupByDate(locationGroup)).toEqual([
+test("mapAidRequestsToFeatures", () => {
+  const mapData = [
     {
-      date: "2022-03-11",
-      aidRequests: [
-        { amount: 10, name: "Sanitary pads" },
-        { amount: 15, name: "Masks" },
-        { amount: 19, name: "Baby Products" },
-        { amount: 16, name: "Medical Kits / Supplies" },
-        { amount: 10, name: "Torches" },
-        { amount: 11, name: "Water" },
-        { amount: 21, name: "Personal hygiene kits" },
-        { amount: 20, name: "Tampons" },
-        { amount: 10, name: "Food" },
-        { amount: 15, name: "Baby Food" },
-      ],
+      city_id: 1,
+      amount: 10,
+      description: "description#1",
     },
     {
-      date: "2022-03-12",
-      aidRequests: [
-        { amount: 4, name: "Personal hygiene kits" },
-        { amount: 4, name: "Medical Kits / Supplies" },
-        { amount: 4, name: "Baby Products" },
-      ],
-    },
-  ]);
-});
-
-test("assignTotalForDate", () => {
-  const grouped = {
-    date: "2022-03-11",
-    aidRequests: [
-      { amount: 10, name: "Sanitary pads" },
-      { amount: 15, name: "Masks" },
-      { amount: 19, name: "Baby Products" },
-      { amount: 16, name: "Medical Kits / Supplies" },
-      { amount: 10, name: "Torches" },
-      { amount: 11, name: "Water" },
-      { amount: 21, name: "Personal hygiene kits" },
-      { amount: 20, name: "Tampons" },
-      { amount: 10, name: "Food" },
-      { amount: 15, name: "Baby Food" },
-    ],
-  };
-
-  expect(assignTotalForDate(grouped).total).toEqual(147);
-});
-
-test("mapAidRequestsToFeatures", () => {
-  const decodedAidRequestGroupedByLocation: DecodedAidRequestGroupedByLocation[] = [
+      city_id: 2,
+      amount: 20,
+      description: "description#2",
+    }
+  ];
+  function locationTranslator(city_id: number) {
```

Comment:
**issue**

Why do you declare a function in a unit test file?

Reply:
it's a mock function used for tests

Reply:
Maybe call it a `mockLocationTranslator`?

Reply:
done